mozilla/bedrock
# Migrate /about/leadership/ page to Wagtail (Fixes #14801)

## One-line summary

Migrates `/about/leadership/` page to Wagtail

## Significant changes and points to review

This does not yet remove the old leadership page / template. That will be done after we have rolled this out and have data in the prod DB.

## Issue / Bugzilla link

#14801

## Testing

> [!NOTE]
> A good test is to use some data from https://www.mozilla.org/en-US/about/leadership/ to create some different sections / groups.

1. Run `make preflight`.
2. Run `npm start`.
3. Auth via http://localhost:8000/cms-admin/.
4. Create an `StructuralPage` with a slug of `about`.
5. Create a child `LeadershipPage` under the `StructuralPage` you created in the prior step, with a slug of `leadership`.
6. Enter some data and preview the page.
7. Publish the page.
8. Go to http://localhost:8000/en-US/about/leadership/, and you should now see the wagtail page instead of the hard-coded page.
